backend/src/Controller/AutocompleteController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AutocompleteController extends AbstractController
+{
+    /**
+     * @Route("/ac/{place}", name="autocomplete", methods={"GET"})
+     */
+    public function index(string $place): JsonResponse
+    {
+        $google_url = "https://maps.googleapis.com/maps/api/place/autocomplete/json?input=$place&key=AIzaSyDZJTQ_zk-aXNr5gH8Si82_AOHiUmVXDJg&language=de";
+
+        $json =  file_get_contents($google_url);
+        $responseBody = json_decode($json);
+
+        header('Content-Type: application/json');
+        header('Access-Control-Allow-Origin: *');
+
+        echo json_encode($responseBody);
```

Comment:
Eigene Klasse "GoogleMapsService"
für das fetchen. 

return JsonResponse
kein json_encode die!

Reply:
I will try that, thank you!